src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -770,3 +774,9 @@ def _parse_drs_uri(self,
                 return str(file_id.path).strip('/')
         else:
             return None
+
+
+def _disallow_joiner(content: str) -> None:
+    reject(config.manifest_column_joiner in content,
+           f'{config.manifest_column_joiner!r} is disallowed',
+           content)
```

Comment:
Consider:

```suggestion
def _disallow_joiner(content: str) -> str:
    reject(config.manifest_column_joiner in content,
           f'{config.manifest_column_joiner!r} is disallowed',
           content)
    return content
```

So you can call the function inline, e.g.

```
links_json['content'] = json.loads(_disallow_joiner(links_json['content']))
```